### DISCUSSION TITLE
$directory not showing full path

### DISCUSSION THREAD

**Discussion by ItsBreeze** [starship/starship#4218] [Q&A]
┌───(Breeze㉿breeze)──[~][08:11 pm]
└─$ cd Documents

┌───(Breeze㉿breeze)──[~][08:11 pm]
└─$ cd ..

the ~ marking the root directory does not update when I switch to a new directory... Is this standard, that I need to change? I don't really understand what is happening or why... I would share my .toml config but this seems to be the standard thing that I need to change bc I haven't changed anything regarding that output? This is also in WSL...

**Comment by chipbuster** [comment]
It depends on what's in your TOML file.

It's also possible that there's some unusual configuration on your system which is causing the starship hooks to not fire correctly. It's rare, but it happens sometimes (usually on unusual shells, or if you've put strange stuff in your init file).

**Reply by ItsBreeze** [reply]
┌───(Breeze㉿Breeze)──[~][12:51 am]
└─$ cd Documents

┌───(Breeze㉿Breeze)──[~\Documents][12:52 am]
└─$
this is the output with the same starship.toml in windows terminal, PowerShell. I believe the issue to not be in the TOML file, but an issue with WSL...

**Comment by davidkna** [comment]
The home inside the WSL environment does not match the home outside of WSL. Is it possible your WSL home does not have a Documents folder?
